src/day-1-toy/11_func.py

```diff
@@ -1,6 +1,15 @@
 # Write a function is_even that will return true if the passed in number is even.
+def even_check(a):
+    return(a%2)==0
 
 # Read a number from the keyboard
 num = input("Enter a number: ")
 
-# Print out "Even!" if the number is even. Otherwise print "Odd"
\ No newline at end of file
+# Print out "Even!" if the number is even. Otherwise print "Odd"
+num = int(4)
```

Comment:
Try commenting this line and inputting a number when you run this program. Remember, the `input()` function returns a _string_ after receiving input from the user. What kind of type does your `even_check()` function need? How can you change the string into that type?